docs/guides/postgres/audit-logging/pgaudit.md

```
@@ -0,0 +1,553 @@
+`SESSION` versus `OBJECT` in the first field tells you which half of the
+configuration produced the record — useful when tuning.
+
+## Use cases, with the records they produce
+
+Each of the following was run against the cluster built above; the log lines are
+the actual output.
+
+### A teller reads a customer's national ID
+
+```bash
+$ psql -U teller -c "SELECT full_name, national_id FROM core.customers WHERE id=1;"
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)